web/views/sleep_tracker_view.ex

```diff
@@ -22,10 +22,35 @@ defmodule Healthlocker.SleepTrackerView do
     if Kernel.length(past_week) == 0 do
       0
     else
-      total_slept / Kernel.length(past_week)
+      format_average_sleep(total_slept / Kernel.length(past_week))
     end
   end
 
+  defp hours_mins(humanized_str) do
+    humanized_str
+      |> String.replace(" hours,", "h")
+      |> String.replace(" minutes", "m")
+      |> String.split(",")
+      |> List.first
+  end
+
+  @doc """
+  format_average_sleep
+
+  ## Example
+
+      iex> import Healthlocker.SleepTrackerView
+      iex> format_average_sleep(7.375)
+      "7h 22m"
+
+  """
+
+  def format_average_sleep(num) do
+    Duration.from_hours(num)
+      |> Timex.Format.Duration.Formatter.format(:humanized)
+      |> hours_mins()
```

Comment:
I _think_ we should be able to use the Timex Formatters for this, the especially [Strftime](https://hexdocs.pm/timex/Timex.Format.DateTime.Formatters.Strftime.html#content) one. We'll have to write our own one and pass it in as the formatter. There are some examples here: https://timex.readme.io/docs/formatting-dates

http://www.foragoodstrftime.com/ is a great resource to test your strftime format.